### PR TITLE
change notification for channel and teams

### PR DESCRIPTION
**Pull request by anandab-msft**

### CODE REVIEW

File: concepts/webhooks.md [hunk 1]
```diff
@@ -294,9 +297,11 @@ The following table lists the latency to expect between an event happening in th
 |:-----|:-----|:-----|
 |[alert][] | Less than 3 minutes | 5 minutes |
 |[callRecord][] | Less than 15 minutes | 60 minutes |
+|[channel][] | Less than 15 minutes | 60 minutes |
```
Comment (abshar-teams) [inline concepts/webhooks.md:300]: should be 10-20 sec, not in minutes

File: concepts/toc.yml [hunk 2]
```diff
@@ -475,6 +475,10 @@ items:
                 href: change-notifications-delivery.md
               - name: Get change notifications for chat and channel messages
                 href: teams-changenotifications-chatmessage.md
+                name: Get change notifications for team and channel
```
Comment (abshar-teams) [inline concepts/toc.yml:478]: Get change notifications for team and channel EVENTS

File: concepts/teams-changenotifications-teamMembership.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
+---
+title: "Get change notifications for any change in Teams membership using Microsoft Graph"
```
Comment (abshar-teams) [inline concepts/teams-changenotifications-teamMembership.md:2]: Get notifications for any membership changes in a team

File: concepts/teams-changenotifications-teamMembership.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+description: "Get change notifications for any change in Teams membership using Microsoft Graph"
+author: "anandab"
+localization_priority: Priority
+ms.prod: "microsoft-teams"
+ms.custom: scenarios:getting-started
+---
+
+# Get change notifications for change in teams membership using Microsoft Graph
+
+Change notifications enable you to subscribe to changes (create, update, and delete) to [membership](/graph/api/resources/conversationmember?preserve-view=true) in a [team](/graph/api/resources/team?preserve-view=true). Change notifications provide a low latency model by allowing you to maintain a [subscription](/graph/api/resources/webhooks?preserve-view=true). You can also get the resource data in the notifications and therefore avoid calling the API to get the payload.
```
Comment (abshar-teams) [inline concepts/teams-changenotifications-teamMembership.md:12]: this paragraph (line 12) shouldn't be required for every page.

File: concepts/teams-changenotifications-teamMembership.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+  "encryptionCertificateId": "{customId}",
+  "expirationDateTime": "2019-09-19T11:00:00.0000000Z",
+  "clientState": "{secretClientState}"
+}
+```
+
+
+## Notification payloads
+
+Depending on your subscription, you can either get the notification with resource data, or without it. Subscribing with resource data allows you to get the message payload along with the notification, removing the need to call back and get the content.
```
Comment (abshar-teams) [inline concepts/teams-changenotifications-teamMembership.md:52]: fix 'message payload'

File: concepts/teams-changenotifications-teamMembership.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+
+### Notifications with resource data
+
+For notifications with resource data, the payload looks like the following. This payload is for a membership change in a team.
```
Comment (abshar-teams) [inline concepts/teams-changenotifications-teamMembership.md:56]: re-phrase

File: concepts/teams-changenotifications-team-and-channels.md [hunk 7]
```diff
@@ -0,0 +1,313 @@
...
+  "encryptionCertificateId": "{customId}",
+  "expirationDateTime": "2019-09-19T11:00:00.0000000Z",
+  "clientState": "{secretClientState}"
+}
+```
+
+
+### Subscribe to changes in any channel at tenant level
+
+To get change notifications for any change in channel's property in any channel or any channel creation or deletion across tenant, subscribe to `teams/getAllChannels`. This resource supports [including resource data](webhooks-with-resource-data.md) in the notification.
```
Comment (abshar-teams) [inline concepts/teams-changenotifications-team-and-channels.md:80]: re-phrase

File: concepts/teams-changenotifications-team-and-channels.md [hunk 8]
```diff
@@ -0,0 +1,313 @@
...
+{
+    "value": [{
+        "subscriptionId": "10493aa0-4d29-4df5-bc0c-ef742cc6cd7f",
+        "changeType": "created",
+        "clientState": "<<--SpecifiedClientState-->>",
+        "subscriptionExpirationDateTime": "2021-02-02T10:30:34.9097561-08:00",
+        "resource": "teams('fb82c19a-0f6d-41ed-90f0-cbb29a476ede')",
+        "resourceData": {
+            "id": "1612289765949",
+            "@odata.type": "#Microsoft.Graph.Teams",
```
Comment (abshar-teams) [inline concepts/teams-changenotifications-team-and-channels.md:159]: its team, not teams

File: concepts/webhooks.md (hunk 1 above)
Comment (anandab-msft) [reply concepts/webhooks.md:300]: @abshar-teams - i changed av. latency to 10-20 sec and max latency to 1 min.

File: concepts/teams-changenotifications-teamMembership.md (hunk 4 above)
Comment (anandab-msft) [reply concepts/teams-changenotifications-teamMembership.md:12]: i reworded this sentence .

File: concepts/teams-changenotifications-teamMembership.md (hunk 5 above)
Comment (anandab-msft) [reply concepts/teams-changenotifications-teamMembership.md:52]: changed it to `membership payload`

File: concepts/teams-changenotifications-team-and-channels.md (hunk 7 above)
Comment (anandab-msft) [reply concepts/teams-changenotifications-team-and-channels.md:80]: rephrased

File: concepts/teams-changenotifications-team-and-channels.md [hunk 9]
```diff
@@ -0,0 +1,313 @@
...
+
+# Get change notifications for team and channel using Microsoft Graph
+
+Change notifications enables you to subscribe to changes in a resource. For more details about creating and managing subscription see [Use the Microsoft Graph API to get change notifications](https://docs.microsoft.com/en-us/graph/api/resources/webhooks?view=preserve-view=true)
+
+>**Note:** The maximum time a subscription can last is 60 minutes; however, subscriptions can be renewed until the caller has permissions to access to resource.
+
+### Subscribe to changes in any team at tenant level
+
+To get change notifications for any change in team's property across tenant, subscribe to `/teams`. This resource supports [including resource data](webhooks-with-resource-data.md) in the notification.
```
Comment (abshar-teams) [inline concepts/teams-changenotifications-team-and-channels.md:18]: this is not just for property updates, but for creation/deletion of any team too

**Pull request closed without merging**